content/docs/2.5/scalers/newrelic.md

```diff
@@ -0,0 +1,70 @@
++++
+title = "Newrelic"
+availability = "2.5+"
+maintainer = "Community"
+description = "Scale applications based on NewRelic NRQL"
+layout = "scaler"
+go_file = "newrelic_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `Newrelic` trigger that scales based on a Newrelic metric.
+
+```yaml
+triggers:
+  - type: newrelic
+    metadata:
+      # Required: nrAccount - Subaccount to run the query on
+      nrAccount: 1234567
+      # Required: nrQueryKey - Api key to connect to New Relic
+      nrQueryKey: "NRAK-xxxxxxxxxxxxxxxxxxxxxxxxxxx"
+      # Required: nrRegion - Region to query data for
+      nrRegion: "US"
+      # Required: metricName
+      metricName: "duration"
+      # Required: nrql
+      nrql: "SELECT average(duration) from Transaction where appName='SITE' TIMESERIES"
+      # Required: threshold
+      threshold: 100
+      # Optional: nrLogLevel
+      nrLogLevel: "info"
+```
+
+**Parameter list:**
+
+- `nrAccount` - The account within Newrelic that the request should be targeted against
```

Comment:
```suggestion
- `nrAccount` - The account within Newrelic that the request should be targeted against.
```

Reply:
This is being based off what @marcelobartsch  has developed so far. I do concur that the NR prefix is not required and would recommend we collective ask to modify the pr and thus need to update the documentation.

Reply:
I've filed a pr against https://github.com/marcelobartsch/keda/pull/1/files branch that is the main code that would remove the nr prefix. Once that is accepted merged into the actual scaler. I'll update the code here to reflect that change as well.